plugins/tenant/acme/config/tenant.yml

```
@@ -0,0 +1,39 @@
+default: &default
+  name: 'ACME Üniversitesi'
+  abbreviation: 'acme'
+  api_host: 'https://api.omu.sh'
+  contact:
+    address: 'ACME Üniversitesi, Kurupelit Kampüsü, Atakum/SAMSUN'
+    phone: '+9 0362 312 1919'
+    country: 'TR'
+    district: 'Atakum'
+  email:
+    domain: 'omu.edu.tr'
+    default_from: 'noreply@baum.omu.edu.tr'
+    support: 'hotline@baum.omu.edu.tr'
+    admin: 'sistem@baum.omu.edu.tr'
+  ldap:
+    organization: omu.edu.tr
+  network:
+    subnet: '10.0.0.0/8'
+  branding:
+    logo:
+      file: 'logos/baum-logo.svg'
```

```suggestion
      file: 'logos/acme-logo.svg'
```
ACME'de olduğumuzu hemen belli eden bir görsel kullansak iyi olacak?  Bu tür görsel uyaranlar önemli.  Slack'te wikipedia media'dan uyarladığım bir logo gönderiyorum (aynı boyutta SVG, acme-logo.svg ismiyle eklenecek)

---

ACME logosu için kullandığım kaynak: https://commons.wikimedia.org/wiki/File:Logo_acme.svg